workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/fragments/group/marketplace-base-fragments/fragments/select-account-dropdown/fragment.json

```
@@ -0,0 +1,9 @@
+{
+	"configurationPath": "configuration.json",
+	"cssPath": "index.css",
+	"htmlPath": "index.html",
+	"icon": "code",
+	"jsPath": "index.js",
+	"name": "select-account-dropdown",
```

```suggestion
	"name": "Account Selection Dropdown",
```

rename the folder to account-selection-dropdown